tests/main/presenters/test_search_results.py

```diff
@@ -39,14 +39,17 @@ def _get_service_result_by_id(self, results, id):
         return False
 
     def setup_method(self, method):
-        super(TestSearchResults, self).setup_method(method)
+        super().setup_method(method)
 
         self.fixture = _get_fixture_data()
         self.multiple_pages_fixture = _get_fixture_multiple_pages_data()
         self._lots_by_slug = framework_helpers.get_lots_by_slug(
             self._get_framework_fixture_data('g-cloud-6')['frameworks']
         )
 
+    def teardown_method(self, method):
```

Comment:
These aren't required to cause `BaseApplicationTest.teardown_method` to run. If it's not defined here `TestSearchResults.teardown_method` will be `BaseApplicationTest.teardown_method`.

```
class A:
    
    def foo():
        pass
    
class B(A):
    pass
    
print(B.foo)

> <function A.foo at 0x1054f2950>

```

Reply:
Ah, good spot - I had blanket-added equivalent `teardown`s where there was a `setup`, but this one didn't end up having any API client patching. Will remove